serial-vault-services-snap/bin/serviceinit.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# Initialize and start the database service
+echo "Initialize the database"
+$SNAP/usr/bin/wrapper-initialize initdb
+$SNAP/usr/bin/wrapper-pg_ctl -D $SNAP_USER_COMMON/data -l $SNAP_USER_COMMON/logs/logfile start
```

Comment:
I've added this to the serviceinit script. Ideally I'd like the serviceinit to run automatically (as a hook?), but I'm not sure if that will be possible.